contracts/child/validator/RewardPool.sol

```diff
@@ -44,7 +48,7 @@ contract RewardPool is IRewardPool, System, Initializable {
         require(totalBlocks != 0, "EPOCH_NOT_COMMITTED");
         uint256 epochSize = validatorSet.EPOCH_SIZE();
         // slither-disable-next-line divide-before-multiply
-        uint256 reward = (baseReward * totalBlocks) / epochSize;
+        uint256 reward = (networkParams.epochReward() * totalBlocks) / epochSize;
```

Comment:
We should change this to disincentivize long epoch times, maybe something like this:
```suggestion
        uint256 epochReward = networkParams.epochReward();
        if (blockSize >= (epochSize * 2)) {
            reward = ((epochReward * totalBlocks) / epochSize) - ((epochReward * (blockSize / epochSize) / 4);
        }
```

Reply:
TBD - added TODO.